angrmanagement/__main__.py

```diff
@@ -3,25 +3,11 @@
 import ctypes
 import threading
 import time
+import warnings
 
-
-def check_dependencies():
-
-    try:
-        import PySide2
-    except ImportError:
-        sys.stderr.write("Cannot find the PySide2 package. You may install it via pip:\n" +
-                         "    pip install pyside2\n")
-        return False
-
-    try:
-        import qtconsole
-    except ImportError:
-        sys.stderr.write("Cannot find the qtconsole package. You may install it via pip:\n" +
-                         "    pip install qtconsole\n")
-        return False
-
-    return True
```

Comment:
As discussed, let's replace it with a proper implementation using `distutils.core.run_setup`.